src/domain.h

```diff
@@ -343,6 +345,37 @@ class Domain : public NLVObject<virDomainPtr, DomainCleanupHandler>
       unsigned int flags_;
   };
 
+  class GetMetadataWorker : public NLVPrimitiveReturnWorker<virDomainPtr, std::string> {
```

Comment:
yeah new fangled compilers let you define whatever you want if you never actually have a callsite for it *grmbl*